app/src/androidTest/java/com/android/sample/ui/activity/CreateActivityScreenTest.kt

```diff
@@ -343,6 +363,9 @@ class CreateActivityScreenTest {
     composeTestRule
         .onNodeWithTag("activityCreateScreen")
         .performScrollToNode(hasTestTag("chooseTypeMenu"))
+
+    composeTestRule.onNodeWithTag("typeTextField").assertIsDisplayed()
+    composeTestRule.onNodeWithText("Activity Type").assertIsDisplayed()
```

Comment:
Here we see that you test for the new test tags display. Great that you added them !